multi-node-k8s.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubectl
+mv kubectl /usr/bin/kubectl
+chmod +x /usr/bin/kubectl
+
+# Vagrant needs libvirtd running
+systemctl start libvirtd
+
+# Log the virsh capabilites so that we know the
+# environment in case something goes wrong.
+virsh capabilities
+
+# TODO: this is not the right way to install vagrant on CentOS, but SCL only
+# provides vagrant 1.x and we need >= 2.2
+yum -y install https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
+yum -y install gcc libvirt-devel libcephfs-devel librbd-devel librados-devel
+
+vagrant plugin install vagrant-libvirt
+
+# setup the kubernes cluster
+git clone https://github.com/galexrt/k8s-vagrant-multi-node
+cd k8s-vagrant-multi-node
+
+make preflight
+make up -j3 BOX_OS=centos VAGRANT_DEFAULT_PROVIDER=libvirt KUBERNETES_VERSION=${K8S_VERSION} MASTER_CPUS=4 MASTER_MEMORY_SIZE_GB=5 NODE_COUNT=2 NODE_MEMORY_SIZE_GB=5 DISK_COUNT=2 DISK_SIZE_GB=25
```

Comment:
Some of these variables should probably be listed in the top of the script, like:

```
WORKERS=2
WORKER_MEM=5
...
make up ... NODE_COUNT=${WORKERS} NODE_MEMORY_SIZE_GB=${WORKER_MEM}
make versions
make status NODE_COUNT=${WORKERS}
```
Having a single place to adjust the values is much nicer

Reply:
Sure. Will make the changes